taxi/urls.py

```diff
@@ -1,9 +1,24 @@
 from django.urls import path
 
-from .views import index
+from taxi.views import index
+from taxi.views import ManufacturerListView
+from taxi.views import CarListView
+from taxi.views import CarDetailView
+from taxi.views import DriverListView
+from taxi.views import DriverDetailView
```

Comment:
Do not duplicate imports, use the following import instead
```suggestion
from taxi.views import (
    index,
    ManufacturerListView,
    CarListView,
    CarDetailView,
    DriverListView,
    DriverDetailView,
)
```